doc/release/1.16.0-notes.rst

```diff
@@ -148,6 +148,12 @@ behavior will be appending.  This applied to: `LDFLAGS`, `F77FLAGS`,
 `F90FLAGS`, `FREEFLAGS`, `FOPT`, `FDEBUG`, and `FFLAGS`.  See gh-11525 for more
 details.
 
+Prefer our include files
+------------------------
+We now prepend the `include/numpy` directory to the `include_dirs` when using
+`nupy.distutils`'s `build_ext` and `build_clib` commands. Previously we appended
+our include dir, which sometimes caused problems when building NumPy.
```

Comment:
Better as "this include directory" rather that "our include dir", to make it clear it's referring to the same one